src/components/Board/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import App from './App'
+import List from './list'
+import Card from './card'
+import Button from './Button'
+
+/**
+~~~~~~~~~~~ List Structure ~~~~~~~~~~~
+list: [{
+    title: "",
+    cards: [{
+        id: "",
+        text: "",
+        moveForward: () => {},
+        moveBackward: () => {},
+    }],
+}],
+}
+**/
+
+export default class Board extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {list: []}
+    }
+
+    render() {
+        return (
+            <h1> {this.props.name} </h1>
```

Comment:
you don't want spaces around `{this.props.name}`, otherwise the spaces will actually show up in the `<h1>` element